internal/packager/fork.go

```diff
@@ -32,38 +32,36 @@ func Fork(originName, forkName, outputDir string, maintainers []string) error {
 
 	// create app dir in output-dir
 	namespace, name := splitPackageName(forkName)
-	appPath := filepath.Join(outputDir, internal.DirNameFromAppName(name))
-	if err := os.MkdirAll(appPath, 0755); err != nil {
+	appDir := filepath.Join(outputDir, internal.DirNameFromAppName(name))
+	if err := os.MkdirAll(appDir, 0755); err != nil {
```

Comment:
I think this `MkdirAll` should be changed to a `Mkdir`. I know it was like this before but it means that you can run `docker-app fork -p /tmp/path/does/not/exist ccrone/hello-world.dockerapp:0.1.0` and it will create all the directories from `/tmp/` onwards.

I would be OK to err out if the parent directory doesn't exist.

@silvin-lubecki WDYT?

Reply:
I think we should do that in a follow-up, so we can focus on it separately.